src/clarity-angular/dropdown/dropdown-submenu.ts

```diff
@@ -0,0 +1,50 @@
+import {Component, ContentChildren, HostListener, Input, QueryList} from "@angular/core";
+import {DropdownItem} from "./dropdown-item";
+
+@Component({
+    selector: "clr-dropdown-submenu",
+    template: `
+        <ng-content></ng-content>
+    `,
```

Comment:
This bothers me. Yet another component to learn about. I forgot to post a precise spec for nested menus before you started working on it, that's on me. We'll talk tomorrow morning and see our options.